File: LeetCode/destinationCity.py
class Solution:
    def destCity(self, paths: List[List[str]]) -> str:

        allpath = []
        toPath = []

        for i in range(0, len(paths)):
            allpath.append(paths[i][0])
            allpath.append(paths[i][1])
            toPath.append(paths[i][0])
            
        for i in range(0, len(allpath)):
            if allpath[i] not in toPath:
                return allpath[i]

File: LeetCode/destinationCity.cpp
#include <bits/stdc++.h>
using namespace std;

class Solution
{
public:
    string destCity(vector<vector<string>> &paths)
    {
        set<string> s;
        vector<string> toPaths;
        for (int i = 0; i < paths.size(); i++)
        {
            s.insert(paths[i][0]);
            s.insert(paths[i][1]);
            toPaths.push_back(paths[i][0]);
        }

        
    }
};
// int main()
// {
//     std::ios::sync_with_stdio(false);
//     cin.tie(nullptr);

//     return 0;
// }